DBConnection/Connection.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Data;
+using System.Data.SqlClient;
+using Logging;
+
+namespace DBConnection
+{
+    public class Connection : Program
+    { 
+        Log Log = new Log();
+        public bool DBConnect()
+        {
+            try
+            { 
+                string sConnectionString = @"Data Source = MyServerName; Initial Catalog = MyDbName; User ID = Admin; Password = Root";
+                SqlConnection sqlconnection = new SqlConnection(sConnectionString);
+                sqlconnection.Open();
+                return true;
+                
+                
+            }
```

Comment:
```suggestion
                sqlconnection.Open();

                return true;
            }
```

Bitte darauf achten keine leeren Lines zu hinterlassen und wenn das "return" statement alleinstehen ist, dann keine Newline davor. Wenn Code davor vorhanden ist, immer leerzeile.